google/cloud/spanner_v1/session.py

```diff
@@ -181,6 +191,7 @@ def exists(self):
         ) as span:
             try:
                 api.get_session(name=self.name, metadata=metadata)
+                self._last_use_time = datetime.now()
```

Comment:
nit: we might not need it here as it is getting assigned in trace_call already.

Reply:
Yeah, you're right. I missed the surrounding trace_call.